judging/scripts/prescribe/generate_prescriptions.py

```diff
@@ -0,0 +1,132 @@
+"""
```

Comment:
This is very copy-pasta with `generate_predictions.py` but I didn't want to refactor (yet) and risk sabotaging the Phase 1 boxes.